pkg/eth/helpers.go

```diff
@@ -40,7 +44,28 @@ func ResolveToNodeType(nodeType int) sdtypes.NodeType {
 }
 
 // ChainConfig returns the appropriate ethereum chain config for the provided chain id
-func ChainConfig(chainID uint64) (*params.ChainConfig, error) {
+func ChainConfig(chainID uint64, chainConfigPath string) (*params.ChainConfig, error) {
+	if chainConfigPath != "" {
```

Comment:
I think it'd be cleaner if we moved this to a separate helper function, e.g. LoadConfig.